erpnext/erpnext_integrations/doctype/letmeship/letmeship.json

```diff
@@ -0,0 +1,55 @@
+{
+ "actions": [],
+ "creation": "2020-07-23 10:55:19.669830",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "enabled",
+  "api_id",
+  "api_password"
+ ],
+ "fields": [
+  {
+   "default": "0",
+   "fieldname": "enabled",
+   "fieldtype": "Check",
+   "label": "Enabled"
+  },
+  {
+   "fieldname": "api_id",
+   "fieldtype": "Data",
+   "label": "API ID",
+   "read_only_depends_on": "eval:doc.enabled == 0"
+  },
+  {
+   "fieldname": "api_password",
+   "fieldtype": "Data",
```

Comment:
API keys should have the field type "Password".

Reply:
Thanks for pointing this out @barredterra. Will also apply the request changes above.